### ISSUE TITLE
display date sent on every message

### ISSUE THREAD

**Issue by cayenne17** [vector-im/element-meta#2277]
#### What would you like to do?
When I browse a communication channel, I only see the time. I haven't figured out how to display the date. I'm often in doubt/frustrated as to whether it's a message from yesterday, today... ?
![image](https://github.com/element-hq/element-desktop/assets/47927025/52fb5124-d52d-49c4-a23c-c4f3942bceb3)

On discord, for example, this is how it's displayed: 
![image](https://github.com/element-hq/element-desktop/assets/47927025/db283a1f-23e2-4ba4-8a44-f542c9570543)

**Comment by t3chguy** [comment]
<img width="668" alt="image" src="https://github.com/element-hq/element-desktop/assets/2403652/d5a0e767-21d2-41cc-8775-2bd2a893f1c6">

**Comment by t3chguy** [comment]
Oh date

**Comment by cayenne17** [comment]
@t3chguy I've just seen an issue that looks pretty much the same on this repository where you've just transferred issue : https://github.com/element-hq/element-meta/issues/1230 ?

**Comment by cayenne17** [comment]
after all, I prefer the way it's presented on Discord to telegram.

**Comment by t3chguy** [comment]
@cayenne17 yes that is similar but not quite the same, its up to the product team which owns this repo to say if they prefer one approach over the other, or offer both

**Comment by claell** [comment]
As we now have #1230 and https://github.com/element-hq/element-web/issues/26925 and https://github.com/element-hq/element-x-android/issues/2270 as application issues and the UI/UX might need a decision, I'll ping @vector-im/product to get some input. Possibly also some designers can decide on this to get a streamlined decision and path forward?
